### ISSUE TITLE
Overwriting files instead of appending -obfuscated..

### ISSUE THREAD

**Issue by YasinY** [javascript-obfuscator/javascript-obfuscator#412]
## Expected Behavior
I'd want the obfuscator to overwrite the files in my /dist/ directory instead of appending an -obfuscated postfix. There has already been a suggestion alike mine, which got declined because it could potentially "lose you code". Fortunately this will not the case for me, as my /dist/ directory is tightly coupled with my /src/ directory since I'm using Typescript to transpile files to dist/*.js. So losing files isn't really applicable.

The issue I've referred to above is here: 
https://github.com/javascript-obfuscator/javascript-obfuscator/issues/306

## Current Behavior
Overwrites files ie. from "lol.js" to "lol-obfuscated.js".

## Your Environment

* Obfuscator version used: 0.18.1

**Comment by slig** [comment]
See #403 .

TL;DR: you better off writing a simple shell script to do that, and you can solve your problem right away.

**Comment by sanex3339** [comment]
My position is not changed, it could potentially "lose you code".
As @zamotkin said, better to write your shell script to do that.

**Comment by GasparZanini** [comment]
Nobody is saying that it is not dangerous @sanex3339, but for certain people like the one that generated this issue @YasinY, for the other person who generated a similar publication and for me it would be beneficial. It is assumed that by using certain tools, one investigates and learns how it works, in addition, everyone works with repositories, even this solution is in a repository, therefore all the changes that one makes are done locally. Did you make a mistake by obfuscating all your files? Nothing happens, you discard changes and/or pull from the repository.

**Comment by tnrich** [comment]
Have to chime in and agree that this would be a very useful feature indeed. Silly not to add it to "protect" peoples codebases. You could even name the variable something like `dangerousOverwriteMode` and add a warning to its usage. 

Really not a valid reason for not including this functionality IMO.

**Comment by tnrich** [comment]
I put in a PR to hopefully solve this issue - https://github.com/javascript-obfuscator/javascript-obfuscator/pull/1305

@sanex3339 I would really appreciate you reconsidering adding this feature. 

Just like how React has dangerouslySetInnerHtml I think exposing this feature with an appropriately named flag is a good way to make sure people know what they're getting into when using it. No one has cause to be upset when their files are overwritten while using a flag that has "dangerous" in the name.
